lib/backblaze/b2/file.rb

```diff
@@ -63,6 +63,10 @@ def create(data:, bucket:, name: nil, base_name: '', content_type: 'b2/x-auto',
         if data.is_a? String
           digest.update(data)
           req.body = data
+        elsif data.is_a? ::Paperclip::UploadedFileAdapter
```

Comment:
Not sure what the best way is to avoid referring to Paperclip::UploadedFileAdapter here. Ideally, the API layer should have no knowledge of Paperclip.